extensions/ql-vscode/src/databases-ui.ts

```diff
@@ -295,7 +295,7 @@ export class DatabaseUI extends DisposableObject {
         'codeQLDatabases.chooseDatabaseLgtm',
         this.handleChooseDatabaseLgtm,
```

Comment:
Optional:
Here is a more involved suggestion for you to try out, which I think would work nicely. Consider it an option for further exploration: you can choose whether to include it as part of this PR, or complete this PR and try this suggestion in a follow-up PR.

If we look at the `handleChooseDatabaseLgtm` function, you can see it has a parameter `progress: ProgressCallback`. Progress callbacks work by allowing you to provide a progress message and step number, e.g. the code
```ts
progress({
  message: 'Choose project',
  step: 1,
  maxStep: 2
})
```
will update the progress notification box with the text `title: message` (e.g. "Adding database from LGTM: Choose project") and a 50% complete progress bar.

Instead of having the "Choose language" text in the title, which you've correctly removed in this PR, I think we could add two `progress()` calls -- one that says "Choose project" and one that says "Choose language". Then the popup box exactly describes the action we are asking the user to carry out in the UI at the top of the screen.

You may need to experiment to choose the best place to put those two progress calls. I think the function `promptImportLgtmDatabase` is a good place for the first one, and `promptForLanguage` is a good place for the second one. Both are reached from `handleChooseDatabaseLgtm`. The second one will require passing a `progress` parameter through to a few more functions.

Reply:
I like this idea! I think for now I'll complete this PR so we can at least have some small improvement, and then I'll spend some time this afternoon and tomorrow working on it.